packages/client/hmi-client/src/workflow/ops/optimize-ciemss/tera-optimize-ciemss.vue

```diff
@@ -187,36 +216,58 @@
 			<Button
 				:disabled="isRunDisabled"
 				outlined
+				severity="secondary"
 				:style="{ marginRight: 'auto' }"
 				label="Run"
 				icon="pi pi-play"
 				@click="runOptimize"
 			/>
-			<div class="label-and-input">
-				<label> Model Config Name</label>
-				<InputText v-model="knobs.modelConfigName" />
-			</div>
-			<div class="label-and-input">
-				<label> Model Config Description</label>
-				<InputText v-model="knobs.modelConfigDesc" />
-			</div>
-			<Button
+			<!-- <Button
```

Comment:
can just delete this button instead of commenting it out. The model configuration modal changes you made look much prettier!